delivery/models.py

```diff
@@ -0,0 +1,47 @@
+from flask import jsonify, request
+from flask.views import MethodView
+from marshmallow import ValidationError
+from delivery.repositories import AbstractRestaurantRepository, MemoryRestaurantRepository
+
+
+class Restaurant:
```

Comment:
and try to use `typing` where it is possible. `typing` is good because:
1. it makes it is easier to read the code
2. it cautions from many dummy errors that would never happen in Java/C++/C# because of their statically typed nature but results in runtime errors in Python e.g. `TypeError`

Reply:
changed